frontend/wallet/src/render/components/Button.re

```diff
@@ -80,6 +80,7 @@ let make =
       ~width=10.5,
       ~height=3.,
       ~icon=?,
+      ~type_="button",
```

Comment:
Are we ever gonna change this? if not we can just remove this line and set `type_="button"` on line 98.

Reply:
it gets changed to "submit" when the form tag is used along with a button we want to trigger the onSubmit function